### PR TITLE
Update the CIP process

### PR DESCRIPTION
**Pull request by oed**
This PR updates the CIP process to make it easier to understand and contribute. The changes encurrages PRs to be created when CIPs are at the draft stage so that they can appear at https://cips.ceramic.network

This PR also changes CIP-1 from "Final" to "Living"

[CIP-1 Preview](https://github.com/ceramicnetwork/CIPs/blob/update-cip-1/CIPs/cip-1.md)

### CODE REVIEW

**Comment by michaelsena** [comment]
lgtm

**Review by michaelsena — APPROVED** [review]
lgtm, worth trying and seeing what works/doesn't

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Bug Report
+description: File a bug report
```
Comment (stbrody) [inline .github/ISSUE_TEMPLATE/bug-report.yml:2]: what is a "bug report" in the context of the CIP repo?  Do we even want this template?

File: README.md [hunk 2]
```diff
@@ -2,89 +2,25 @@
...
-- `Final (non-Core)`: a non-core CIP that has met all criteria and is finished.
+1. Review [CIP-1](https://cips.ceramic.network/CIPs/cip-1).
+2. Fork the repository.
+3. Add your CIP to your fork of the repository. There is a [template CIP here](https://github.com/ceramicnetwork/CIPs/blob/main/cip-template.md).
+4. Submit a Pull Request.
 
 # Editors
 - Michael Sena ([@michaelsena](http://github.com/michaelsena))
 - Joel Thorstensson ([@oed](http://github.com/oed))
 - Spencer T Brody ([@stbrodfy](http://github.com/stbrody))
```
Comment (stbrody) [inline README.md:25]:
```suggestion
- Spencer T Brody ([@stbrody](http://github.com/stbrody))
```

**Review by stbrody — APPROVED** [review]
lgtm

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (oed) [reply .github/ISSUE_TEMPLATE/bug-report.yml:2]: Automations being broken.

**Pull request merged**